tests/data/mstool/output/AllergyIntolerance.ndjson

```diff
@@ -0,0 +1,30 @@
+{
+  "resourceType" : "AllergyIntolerance",
+  "meta": {
+    "security": [{
+      "code": "REDACTED",
+      "display": "redacted",
+      "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationValue"
+    }]
+  },
```

Comment:
mostly for my own edification: how does this work on the MS side?

Reply:
Ah OK for context: this file is the expected output of the MS tool. It's input counterpart is in the `input/` folder.

To your actual question: this bit here is inserted by the MS tool automatically (can't turn it off). I find this annoying, and pointless for Cumulus' purposes, so the ETL drops this field again when processing files.

But this unit test doesn't do the full ETL processing, it's just testing our ms-config.json file, so the field is still present.